src/client/components/EducationHistoryTile.js

```diff
@@ -13,8 +13,16 @@ const EducationHistoryTile = ({
 }) => (
   <SingleEducationItem>
     <div>
-      <Text.Block size="large">{educationHistoryData.organization}</Text.Block>
-      <Text.Block size="normal">
+      <Text.Block
+        size="large"
+        css={`
+          line-height: 1.8;
+          font-weight: bold;
```

Comment:
Do we need this?

Reply:
That made a subtle difference as requested, yes, both lines.